spec/integ/matrix-client-event-timeline.spec.ts

```diff
@@ -711,6 +752,63 @@ describe("MatrixClient event timelines", function() {
     });
 
     describe("getLatestTimeline", function() {
```

Comment:
We should also add some tests for the thread scenarios added to `getLatestTimeline(...)` in this PR.

It was originally only intended for the main live timeline (`room.getUnfilteredTimelineSet(...)`) and our only usage in the SDK from `room.refreshLiveTimeline(...)` is still for that purpose